### PR TITLE
Initial skeleton for the helm chart (not functional)

### PR DESCRIPTION
**Pull request by jlpedrosa**
This is the basic structure for a helm chart, still not even tried to run it as we don't have the docker image published (yet)

### CODE REVIEW

File: charts/mikrotik-prom-exporter/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
+version: 0.1.0
+
+# This is the version number of the application being deployed. This version number should be
+# incremented each time you make changes to the application. Versions are not expected to
+# follow Semantic Versioning. They should reflect the version the application is using.
+# It is recommended to use it with quotes.
+appVersion: "1.16.0"
```
Comment (vaerh) [inline charts/mikrotik-prom-exporter/Chart.yaml:24]: M.b. 1.0.0?

File: charts/mikrotik-prom-exporter/templates/NOTES.txt [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+{{- else if contains "LoadBalancer" .Values.service.type }}
+     NOTE: It may take a few minutes for the LoadBalancer IP to be available.
+           You can watch the status of by running 'kubectl get --namespace {{ .Release.Namespace }} svc -w {{ include "mikrotik-prom-exporter.fullname" . }}'
+  export SERVICE_IP=$(kubectl get svc --namespace {{ .Release.Namespace }} {{ include "mikrotik-prom-exporter.fullname" . }} --template "{{"{{ range (index .status.loadBalancer.ingress 0) }}{{.}}{{ end }}"}}")
+  echo http://$SERVICE_IP:{{ .Values.service.port }}
+{{- else if contains "ClusterIP" .Values.service.type }}
+  export POD_NAME=$(kubectl get pods --namespace {{ .Release.Namespace }} -l "app.kubernetes.io/name={{ include "mikrotik-prom-exporter.name" . }},app.kubernetes.io/instance={{ .Release.Name }}" -o jsonpath="{.items[0].metadata.name}")
+  export CONTAINER_PORT=$(kubectl get pod --namespace {{ .Release.Namespace }} $POD_NAME -o jsonpath="{.spec.containers[0].ports[0].containerPort}")
+  echo "Visit http://127.0.0.1:8080 to use your application"
+  kubectl --namespace {{ .Release.Namespace }} port-forward $POD_NAME 8080:$CONTAINER_PORT
```
Comment (vaerh) [inline charts/mikrotik-prom-exporter/templates/NOTES.txt:21]: 8080 or 9100?

Comment (jlpedrosa) [reply charts/mikrotik-prom-exporter/templates/NOTES.txt:21]: TBH, prom traffic is http, probably we should move it to 8080 or 80. In any case, this template does not work at all, we don't even have the docker image.

File: charts/mikrotik-prom-exporter/Chart.yaml (hunk 1 above)
Comment (jlpedrosa) [reply charts/mikrotik-prom-exporter/Chart.yaml:24]: or 0.0.1 :rofl:  The idea is that the number will be updated automatically in CI with the releases....

File: charts/mikrotik-prom-exporter/templates/NOTES.txt (hunk 2 above)
Comment (vaerh) [reply charts/mikrotik-prom-exporter/templates/NOTES.txt:21]: I made my first approach to building binaries yesterday. Until I finish with it - nothing to put in the docker container.
By default, the exporter server listens on port 9100 and the container opens port 9100 too. That's the reason I asked.

Comment (jlpedrosa) [reply charts/mikrotik-prom-exporter/templates/NOTES.txt:21]: I mean I think it's more reasonable to move the containter to 8080....

**Pull request merged**